src/olympia/devhub/templates/devhub/versions/list.html

```diff
@@ -87,7 +97,7 @@
          data-is-current="{{ (version == addon.current_version)|int }}">x</a>
     </td>
   </tr>
-  {% if full_info and version.channel == amo.CHANNEL_LISTED and (can_request_review or can_cancel) and check_addon_ownership(request.user, addon, allow_developer=True) %}
```

Comment:
The view is already decorated by `dev_required` which calls `check_addon_ownership(request.user, addon, allow_developer=True, allow_mozilla_disabled_addon)`. `can_cancel` and `can_request_review` will never be `True` if the add-on is disabled, so this extra check is not needed.